MSYS build depends on libwinpthread-1.dll

### Describe your bug here. And how to reproduce it.

Since commit 3be6ece until most recent 3b7f2a6 the compilation using MSYS is broken.

For windows target the error is:
```cd tables; /usr/bin/env python3 compile_resources.py ../tables/zelda3.sfc
In file included from third_party/opus-1.3.1-stripped/pitch.h:42,
                 from third_party/opus-1.3.1-stripped/bands.c:44,
                 from third_party/opus-1.3.1-stripped/opus_decoder_amalgam.c:11:
third_party/opus-1.3.1-stripped/x86/pitch_sse.h:37:10: fatal error: config.h: No such file or directory
   37 | #include "config.h"
      |          ^~~~~~~~~~
compilation terminated.
make: *** [Makefile:17: third_party/opus-1.3.1-stripped/opus_decoder_amalgam.o] Error 1
make: *** Waiting for unfinished jobs....
```


for Switch Target the error is:
```c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find tracing.o: No such file or directory
collect2.exe: error: ld returned 1 exit status
make[1]: *** [/opt/devkitpro/libnx/switch_rules:80: /home/Impeeza/zelda3/platform/switch/zelda3.elf] Error 1
make: *** [Makefile:161: bin] Error 2
```


The packages installed are:
- git
- make
- mingw-w64-x86_64-gcc
- mingw-w64-x86_64-pkgconf
- mingw-w64-x86_64-python-yaml
- mingw-w64-x86_64-python3-pillow
- mingw-w64-x86_64-SDL2
- switch-dev
- switch-sdl2
- switch-tools

going back to commit ec08202 all compile right

Using TCC the file compiles OK.

### What is your build target?

Windows

MinGW64 seems to build fine, but MinGW32 throws some wild new errors:

```
cc -c -O2 -Werror -IC:/msys64/mingw32/include/SDL2 -Dmain=SDL_main -DSYSTEM_VOLUME_MIXER_AVAILABLE=0 third_party/opus-1.3.1-stripped/opus_decoder_amalgam.c -o third_party/opus-1.3.1-stripped/opus_decoder_amalgam.o
In file included from third_party/opus-1.3.1-stripped/opus_decoder_amalgam.c:24:
third_party/opus-1.3.1-stripped/x86/pitch_sse.c: In function 'dual_inner_prod_sse':
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:86:10: error: SSE vector return without SSE enabled
changes the ABI [-Werror=psabi]
   86 |    xsum1 = _mm_setzero_ps();
      |    ~~~~~~^~~~~~~~~~~~~~~~~~
In file included from third_party/opus-1.3.1-stripped/x86/pitch_sse.c:40:
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:118:1: error: inlining failed
in call to 'always_inline' '_mm_setzero_ps': target specific option mismatch
  118 | _mm_setzero_ps (void)
      | ^~~~~~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:87:12: note: called from here
   87 |    xsum2 = _mm_setzero_ps();
      |            ^~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:118:1: error: inlining failed
in call to 'always_inline' '_mm_setzero_ps': target specific option mismatch
  118 | _mm_setzero_ps (void)
      | ^~~~~~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:86:12: note: called from here
   86 |    xsum1 = _mm_setzero_ps();
      |            ^~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:184:1: error: inlining failed
in call to 'always_inline' '_mm_add_ps': target specific option mismatch
  184 | _mm_add_ps (__m128 __A, __m128 __B)
      | ^~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:94:15: note: called from here
   94 |       xsum2 = _mm_add_ps(xsum2,_mm_mul_ps(xi, y2i));
      |               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:196:1: error: inlining failed
in call to 'always_inline' '_mm_mul_ps': target specific option mismatch
  196 | _mm_mul_ps (__m128 __A, __m128 __B)
      | ^~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:94:15: note: called from here
   94 |       xsum2 = _mm_add_ps(xsum2,_mm_mul_ps(xi, y2i));
      |               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:184:1: error: inlining failed
in call to 'always_inline' '_mm_add_ps': target specific option mismatch
  184 | _mm_add_ps (__m128 __A, __m128 __B)
      | ^~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:93:15: note: called from here
   93 |       xsum1 = _mm_add_ps(xsum1,_mm_mul_ps(xi, y1i));
      |               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:196:1: error: inlining failed
in call to 'always_inline' '_mm_mul_ps': target specific option mismatch
  196 | _mm_mul_ps (__m128 __A, __m128 __B)
      | ^~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:93:15: note: called from here
   93 |       xsum1 = _mm_add_ps(xsum1,_mm_mul_ps(xi, y1i));
      |               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:932:1: error: inlining failed
in call to 'always_inline' '_mm_loadu_ps': target specific option mismatch
  932 | _mm_loadu_ps (float const *__P)
      | ^~~~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:92:20: note: called from here
   92 |       __m128 y2i = _mm_loadu_ps(y02+i);
      |                    ^~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:932:1: error: inlining failed
in call to 'always_inline' '_mm_loadu_ps': target specific option mismatch
  932 | _mm_loadu_ps (float const *__P)
      | ^~~~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:91:20: note: called from here
   91 |       __m128 y1i = _mm_loadu_ps(y01+i);
      |                    ^~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:932:1: error: inlining failed
in call to 'always_inline' '_mm_loadu_ps': target specific option mismatch
  932 | _mm_loadu_ps (float const *__P)
      | ^~~~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:90:19: note: called from here
   90 |       __m128 xi = _mm_loadu_ps(x+i);
      |                   ^~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:961:1: error: inlining failed
in call to 'always_inline' '_mm_store_ss': target specific option mismatch
  961 | _mm_store_ss (float *__P, __m128 __A)
      | ^~~~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:102:4: note: called from here
  102 |    _mm_store_ss(xy2, xsum2);
      |    ^~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:128:1: error: inlining failed
in call to 'always_inline' '_mm_add_ss': target specific option mismatch
  128 | _mm_add_ss (__m128 __A, __m128 __B)
      | ^~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:101:12: note: called from here
  101 |    xsum2 = _mm_add_ss(xsum2, _mm_shuffle_ps(xsum2, xsum2, 0x55));
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:744:1: error: inlining failed
in call to 'always_inline' '_mm_shuffle_ps': target specific option mismatch
  744 | _mm_shuffle_ps (__m128 __A, __m128 __B, int const __mask)
      | ^~~~~~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:101:12: note: called from here
  101 |    xsum2 = _mm_add_ss(xsum2, _mm_shuffle_ps(xsum2, xsum2, 0x55));
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:184:1: error: inlining failed
in call to 'always_inline' '_mm_add_ps': target specific option mismatch
  184 | _mm_add_ps (__m128 __A, __m128 __B)
      | ^~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:100:12: note: called from here
  100 |    xsum2 = _mm_add_ps(xsum2, _mm_movehl_ps(xsum2, xsum2));
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:785:1: error: inlining failed
in call to 'always_inline' '_mm_movehl_ps': target specific option mismatch
  785 | _mm_movehl_ps (__m128 __A, __m128 __B)
      | ^~~~~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:100:12: note: called from here
  100 |    xsum2 = _mm_add_ps(xsum2, _mm_movehl_ps(xsum2, xsum2));
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:961:1: error: inlining failed
in call to 'always_inline' '_mm_store_ss': target specific option mismatch
  961 | _mm_store_ss (float *__P, __m128 __A)
      | ^~~~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:99:4: note: called from here
   99 |    _mm_store_ss(xy1, xsum1);
      |    ^~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:128:1: error: inlining failed
in call to 'always_inline' '_mm_add_ss': target specific option mismatch
  128 | _mm_add_ss (__m128 __A, __m128 __B)
      | ^~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:98:12: note: called from here
   98 |    xsum1 = _mm_add_ss(xsum1, _mm_shuffle_ps(xsum1, xsum1, 0x55));
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:744:1: error: inlining failed
in call to 'always_inline' '_mm_shuffle_ps': target specific option mismatch
  744 | _mm_shuffle_ps (__m128 __A, __m128 __B, int const __mask)
      | ^~~~~~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:98:12: note: called from here
   98 |    xsum1 = _mm_add_ss(xsum1, _mm_shuffle_ps(xsum1, xsum1, 0x55));
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:184:1: error: inlining failed
in call to 'always_inline' '_mm_add_ps': target specific option mismatch
  184 | _mm_add_ps (__m128 __A, __m128 __B)
      | ^~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:97:12: note: called from here
   97 |    xsum1 = _mm_add_ps(xsum1, _mm_movehl_ps(xsum1, xsum1));
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/msys64/mingw32/lib/gcc/i686-w64-mingw32/12.2.0/include/xmmintrin.h:785:1: error: inlining failed
in call to 'always_inline' '_mm_movehl_ps': target specific option mismatch
  785 | _mm_movehl_ps (__m128 __A, __m128 __B)
      | ^~~~~~~~~~~~~
third_party/opus-1.3.1-stripped/x86/pitch_sse.c:97:12: note: called from here
   97 |    xsum1 = _mm_add_ps(xsum1, _mm_movehl_ps(xsum1, xsum1));
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1.exe: all warnings being treated as errors
make: *** [Makefile:17: third_party/opus-1.3.1-stripped/opus_decoder_amalgam.o] Error 1
```

You could use the system provided packages via `pacman -S mingw-w64-x86_64-opus` and `pacman -S mingw-w64-i686-opus` for 64 bit and 32 bit

Hi, Windows version compiled with MSYS have a dependency:
![image](https://user-images.githubusercontent.com/41979604/196600713-d51d7b99-392b-47d8-9fff-ab99322a0496.png)
but works fine.

The Switch target do not compile the error:

```
linking zelda3.elf
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find attract.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find ending.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find hud.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find misc.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find overlord.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find player_oam.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find poly.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find select_file.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find spc_player.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find sprite_main.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find tagalong.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find tile_detect.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find apu.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find cart.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find cpu.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find dma.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find dsp.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find input.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find snes.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find snes_other.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find spc.o: No such file or directory
c:/msys64/opt/devkitpro/devkita64/bin/../lib/gcc/aarch64-none-elf/12.1.0/../../../../aarch64-none-elf/bin/ld.exe: cannot find tracing.o: No such file or directory
collect2.exe: error: ld returned 1 exit status
make[1]: *** [/opt/devkitpro/libnx/switch_rules:80: /home/Impeeza/zelda3/platform/switch/zelda3.elf] Error 1
make: *** [Makefile:161: bin] Error 2
```
what I am missing?

--- a/Makefile
+++ b/Makefile
@@ -4,25 +4,38 @@ SRCS:=$(wildcard *.c snes/*.c) third_party/gl_core/gl_core_3_1.c third_party/opu
 OBJS:=$(SRCS:%.c=%.o)
 PYTHON:=/usr/bin/env python3
 CFLAGS:=$(if $(CFLAGS),$(CFLAGS),-O2 -Werror)
-
 CFLAGS:=${CFLAGS} $(shell sdl2-config --cflags) -DSYSTEM_VOLUME_MIXER_AVAILABLE=0
-LDFLAGS:=${LDFLAGS} $(shell sdl2-config --libs) -lm
+
+ifeq (${OS},Windows_NT)
+    WINDRES:=windres
+    RES:=zelda3.res
+    LDFLAGS:=${LDFLAGS} -Wl,-Bstatic $(shell sdl2-config --static-libs) -lm
+else
+    LDFLAGS:=${LDFLAGS} $(shell sdl2-config --libs) -lm
+endif
 
 .PHONY: all clean clean_obj clean_gen
 
 all: $(TARGET_EXEC) tables/zelda3_assets.dat
-$(TARGET_EXEC): $(OBJS)
-	$(CC) $(OBJS) -o $@ $(LDFLAGS)
+$(TARGET_EXEC): $(OBJS) $(RES)
+	$(CC) $^ -o $@ $(LDFLAGS)
 %.o : %.c
 	$(CC) -c $(CFLAGS) $< -o $@
 
+$(RES): platform/win32/zelda3.rc
+	@echo "Generating Windows resources"
+	@$(WINDRES) $< -O coff -o $@
+
 tables/zelda3_assets.dat: tables/dialogue.txt
-	cd tables; $(PYTHON) compile_resources.py ../$(ROM)
+	@echo "Compiling game resources"
+	@cd tables; $(PYTHON) compile_resources.py ../$(ROM)
 tables/dialogue.txt:
-	cd tables; $(PYTHON) extract_resources.py ../$(ROM)
+	@echo "Extracting game resources"
+	@cd tables; $(PYTHON) extract_resources.py ../$(ROM)
 
 clean: clean_obj clean_gen
 clean_obj:
-	$(RM) $(OBJS) $(TARGET_EXEC)
+	@$(RM) $(OBJS) $(TARGET_EXEC)
 clean_gen:
-	$(RM) tables/zelda3_assets.dat
+	@$(RM) $(RES) tables/zelda3_assets.dat tables/*.txt tables/*.png tables/sprites/*.png tables/*.yaml
+	@rm -rf tables/__pycache__ tables/dungeon tables/img tables/overworld tables/sound

--- a/Makefile
+++ b/Makefile
@@ -9,16 +9,16 @@ CFLAGS:=${CFLAGS} $(shell sdl2-config --cflags) -DSYSTEM_VOLUME_MIXER_AVAILABLE=
 ifeq (${OS},Windows_NT)
     WINDRES:=windres
     RES:=zelda3.res
-    LDFLAGS:=${LDFLAGS} -Wl,-Bstatic $(shell sdl2-config --static-libs) -lm
+    SDLFAGS:=${LDFLAGS} -Wl,-Bstatic $(shell sdl2-config --static-libs) -lm
 else
-    LDFLAGS:=${LDFLAGS} $(shell sdl2-config --libs) -lm
+    SDLFLAGS:=${LDFLAGS} $(shell sdl2-config --libs) -lm
 endif
 
 .PHONY: all clean clean_obj clean_gen
 
 all: $(TARGET_EXEC) tables/zelda3_assets.dat
 $(TARGET_EXEC): $(OBJS) $(RES)
-	$(CC) $^ -o $@ $(LDFLAGS)
+	$(CC) $^ -o $@ $(LDFLAGS) $(SDLFLAGS)
 %.o : %.c
 	$(CC) -c $(CFLAGS) $< -o $@
 

--- a/Makefile
+++ b/Makefile
@@ -9,9 +9,9 @@ CFLAGS:=${CFLAGS} $(shell sdl2-config --cflags) -DSYSTEM_VOLUME_MIXER_AVAILABLE=
 ifeq (${OS},Windows_NT)
     WINDRES:=windres
     RES:=zelda3.res
-    SDLFAGS:=${LDFLAGS} -Wl,-Bstatic $(shell sdl2-config --static-libs) -lm
+    SDLFLAGS:=-Wl,-Bstatic $(shell sdl2-config --static-libs)
 else
-    SDLFLAGS:=${LDFLAGS} $(shell sdl2-config --libs) -lm
+    SDLFLAGS:=$(shell sdl2-config --libs) -lm
 endif
 
 .PHONY: all clean clean_obj clean_gen

--- a/glsl_shader.c
+++ b/glsl_shader.c
@@ -7,6 +7,7 @@
 #include <assert.h>
 #include <string.h>
 #define STB_IMAGE_IMPLEMENTATION
+#define STBI_NO_THREAD_LOCALS
 #define STBI_ONLY_PNG
 #define STBI_MAX_DIMENSIONS 4096
 #define STBI_NO_FAILURE_STRINGS

